src/app/containers/Header/index.test.jsx

```diff
@@ -92,4 +92,15 @@ describe(`Header`, () => {
     );
     expect(document.querySelector(`header nav`)).toBeNull();
   });
+
+  it('should render a Brand with a Skip to content link, linking to #content', () => {
+    render(
+      HeaderContainerWithContext({
+        pageType: 'frontPage',
+      }),
+    );
+
+    const skipLink = document.querySelectorAll("a[href='#content']")[1];
```

Comment:
This test passes but doesn't actually test the condition (I copied it incorrectly from the Brand container) - am currently fixing